hack/update-codegen.sh

```diff
@@ -4,36 +4,42 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# Expected performance drop after moving to go mod
-# see https://github.com/kubernetes/code-generator/issues/69
-# TODO: fix it
+REPO_ROOT="${REPO_ROOT:-$(git rev-parse --show-toplevel)}"
+export GO111MODULE="on"
+go mod vendor
```

Comment:
`go mod vendor` has been done in Makefile, put it here to make sure `./hack/update-codegen.sh` also works.